index.html

```diff
@@ -2373,18 +2373,20 @@ <h2 class="header">Year</h2>
 				<h2>Wizard</h2>
 				<div class="thin-box">
 					<div class="wizard" data-initialize="wizard" id="myWizard">
-						<ul class="steps">
-							<li data-step="1"><span class="badge">1</span>Campaign<span class="chevron"></span>
-							</li>
-							<li class="active" data-step="2"><span class="badge">2</span>Recipients<span class="chevron"></span>
-							</li>
-							<li data-step="3"><span class="badge">3</span>Template<span class="chevron"></span>
-							</li>
-							<li data-step="4"><span class="badge">4</span>Preview<span class="chevron"></span>
-							</li>
-							<li data-step="5" data-name="distep"><span class="badge">5</span>Send<span class="chevron"></span>
-							</li>
-						</ul>
+						<div class="steps">

```

Comment:
instead of moving steps class (causing js selectors to need to be update) please add a new class